src/frontend/src/components/AddUser/index.tsx

```diff
@@ -46,18 +82,56 @@ class AddUser extends React.Component<Props, State> {
     event.preventDefault();
 
     // Standardize email-lowerCase and trim both form inputs
+        // Standardize name-lowerCase
     this.setState(
       {
+        name: this.state.name.toLowerCase().trim(),
```

Comment:
We should leave the `name` field as case-sensitive, so this `toLowerCase()` can be removed.

Reply:
Ok I blew away the master adn synced with upstream. I made new branch, added the changes, and pushed it to my repo.  I'll make a pr from there. Should I close this pr?

Reply:
And thanks for the help

Reply:
Yep, you can close this PR.